fault_tolerance_slave.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+"""
+Copyright 2019 Marco Lattuada
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import argparse
+import os
+import sys
+
+import sequence_data_processing
+
+
+def generate_test():
+    test = {
```

Comment:
You should add a brief docstring like "Generates a dictionary with several models and configurations"